unitTests/utils/SearchboxResizeTest.ts

```diff
@@ -0,0 +1,81 @@
+import * as Mock from '../MockEnvironment';
+import { Searchbox } from '../../src/ui/Searchbox/Searchbox';
+import { ISearchboxOptions } from '../../src/ui/Searchbox/Searchbox';
+import { $$ } from '../../src/utils/Dom';
+import _ = require('underscore');
+
+function findHtmlElement(element: HTMLElement, HTMLElements) {
```

Comment:
You could create a Typescript interface for that struct and just return the object
```
return {
   CoveoOmniboxOrQuery: $$(element).find('.CoveoOmnibox') || $$(element).find('.CoveoQuerybox'),
   ...
}
```